floor_generator/lib/generator.dart

```diff
@@ -26,8 +26,9 @@ class FloorGenerator extends GeneratorForAnnotation<annotations.Database> {
     final daoGetters = database.daoGetters;
 
     final databaseClass = DatabaseWriter(database).write();
-    final daoClasses =
-        daoGetters.map((daoGetter) => DaoWriter(daoGetter.dao).write());
+    final daoClasses = daoGetters.map((daoGetter) => DaoWriter(
+            daoGetter.dao, database.entityStreams, database.hasViewStreams)
```

Comment:
the fix part 2: All entitystreams and viewstream indicators of the database(=all DAOs) are given to the DaoWriter, instead of the DaoWriter only considering the entitystreams and viewstreams of its own Dao